source/ThickThinTime.mc

```diff
@@ -4,14 +4,20 @@ using Toybox.Application as App;
 
 class ThickThinTime extends Ui.Drawable {
 
+	var mTimeZones;
+
 	private var mThemeColour, mBackgroundColour;
 	private var mHoursFont, mMinutesFont, mSecondsFont;
+	private var mhalfDCHeight, mhalfDCWidth, mAMPMposition;
```

Comment:
It can be tricky to know what to cache as an instance variable: these collectively can consume a surprising amount of runtime memory. In the recent round of memory improvements, I've actually removed several instance variables, particularly where the values can easily be calculated on each draw cycle, and stored temporarily on the stack.